tests/libvmi/cloudinit.go

```diff
@@ -71,6 +71,17 @@ func WithCloudInitConfigDriveUserData(data string) Option {
 	}
 }
 
+// WithCloudInitConfigDriveEncodedUserData adds cloud-init config-drive with encoded user data.
+func WithCloudInitConfigDriveEncodedUserData(data string) Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		addDiskVolumeWithCloudInitConfigDrive(vmi, cloudInitDiskName, v1.DiskBusVirtio)
+
+		volume := getVolume(vmi, cloudInitDiskName)
+		volume.CloudInitConfigDrive.UserData = ""
+		volume.CloudInitConfigDrive.UserDataBase64 = data
```

Comment:
How could this pass? You are providing a non-encoded data here.

(BTW I don't understand the point of the encoded field, and would vote to stop testing it in e2e)

Reply:
My bad I pushed the wrong changes, PTAL